notebooks/official/pipelines/google_cloud_pipeline_components_automl_tabular.ipynb

```diff
@@ -353,9 +353,9 @@
         "id": "timestamp"
```

Comment:
<p>Add a heading to the cell -&gt; Check installed package versions</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/843/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>